README.md

```diff
@@ -4,7 +4,18 @@ Django web app of the laboîte project http://laboite.cc/help
 ## Getting Started
 
 Make sure you are using a virtual environment of some sort (e.g. `virtualenv` or
-`pyenv`).
+`pyenv`) and the following packages are installed :
+pytz
+jsonfield
+django-openid-auth
+libxslt1-dev
+python3-dev
+zlib1g-dev
+libjpeg-dev
+
+for exemple on linux distributions, use :
```

Comment:
Maybe start the sentence with an uppercase letter? Also, in english, there's no space before a colon.